lib/addons/paapi.ts

```diff
@@ -58,51 +58,52 @@ function elementInnerSize(element: HTMLElement): Size {
 }
 
 /*
- * installGPTSlotAuctionConfig obtains the auction configuration for the current origin
- * and installs it into the given GPT slot.
+ * installGPTAuctionConfigs obtains the auction configuration for the current origin
+ * and installs it into the page GPT slots, optionally filtered to a given domID.
  */
-OptableSDK.prototype.installGPTSlotAuctionConfig = async function(domID: string, defaults: Partial<AuctionConfig> = {}): Promise<void> {
+type GPTAuctionConfigsFilters = {
+  domIDs?: string[];
```

Comment:
Just make this a lambda func(Slot) boolean, this API will be future proof and you can easily implement the desired feature (slotid IN list).

Reply:
Fixed